src/components/svgs/nodes/chengdumetro-line-badge.tsx

```diff
@@ -0,0 +1,181 @@
+        [id, handlePointerMove]
+    );
+    const onPointerUp = React.useCallback(
+        (e: React.PointerEvent<SVGElement>) => handlePointerUp(id, e),
+        [id, handlePointerUp]
+    );
+
+    const fgColor = color[3];
+    const fontSize = 16;
+
+    return (
+        <g
+            id={id}
+            transform={`translate(${x - 12.5}, ${y - 12.5})`}
+            onPointerDown={onPointerDown}
+            onPointerMove={onPointerMove}
+            onPointerUp={onPointerUp}
+            style={{ cursor: 'move' }}
+        >
+            {type == 'normal' ? (
+                <>
+                    <rect fill={color[2]} x="0" width="25" height="25" rx="12.5" ry="12.5" />
+                    <text
+                        className="rmp-name__zh"
+                        textAnchor="middle"
+                        x="12.5"
+                        y="12.5"
+                        fill={fgColor}
+                        fontSize={fontSize}
+                        dominantBaseline="central"
+                    >
+                        {num}
+                    </text>
+                </>
+            ) : type == 'suburban' ? (
+                <>
+                    <rect fill={color[2]} x="0" y="0" width="20" height="25" rx="0" ry="0" />
+                    <rect fill={color[2]} x="19" y="0" width="6" height="5" rx="0" ry="0" />
+                    <rect fill={color[2]} x="19" y="10" width="6" height="5" rx="0" ry="0" />
+                    <rect fill={color[2]} x="19" y="20" width="6" height="5" rx="0" ry="0" />
+                    <text
+                        className="rmp-name__zh"
+                        textAnchor="left"
+                        x="0"
+                        y="12.5"
+                        fill={fgColor}
+                        fontSize={fontSize}
+                        dominantBaseline="central"
+                        style={{ fontFamily: '"Microsoft YaHei","Sogoe UI",Arial,sans-serif' }}
+                    >
+                        {num}
+                    </text>
+                </>
+            ) : (
+                <>
+                    <rect fill={color[2]} x="-1.25" y="0" width="27.5" height="7.5" rx="0" ry="0" />
+                    <rect fill={color[2]} x="7.5" y="0" width="10" height="25" rx="0" ry="0" />
+                    <MultilineText
+                        text={num.toString().split('')}
+                        lineHeight={10.5}
+                        x={12.5}
+                        y={6.5}
+                        fill="white"
+                        fontSize={10}
+                        textAnchor="middle"
+                        grow="down"
+                        dominantBaseline="middle"
+                    />
+                </>
+            )}
+        </g>
+    );
+};
+
+/**
+ * ChengduMetroLineBadge specific props.
+ */
+export interface ChengduMetroLineBadgeAttributes extends AttributesWithColor {
+    num: number | string;
```

Comment:
This key is misleading. Either be `content: number | string` or 

```
num: number;
text: string;
```

and use one of the value by `type`.